release-process.md

```diff
@@ -283,11 +283,14 @@ which fetches potential replacements from Github and JIRA. For instance:
 ```
 $ cd release-spark/dev/create-release
 # Set RELEASE_TAG and PREVIOUS_RELEASE_TAG
-$ vim generate-contributors.py
+$ export RELEASE_TAG=v1.1.1
+$ export PREVIOUS_RELEASE_TAG=v1.1.0
 # Generate initial contributors list, likely with warnings
 $ ./generate-contributors.py
-# Set JIRA_USERNAME, JIRA_PASSWORD, and GITHUB_API_TOKEN
-$ vim release-spark/dev/translate-contributors.py
+# set JIRA_USERNAME, JIRA_PASSWORD, and GITHUB_API_TOKEN
+$ export JIRA_USERNAME=blabla
+$ export JIRA_PASSWORD=blabla
+$ export GITHUB_API_TOKEN=blabla
```

Comment:
Do we require this?

Reply:
I think so. It needs to access github data to get a people's full name.

Reply:
I see. Got it.